.changeset/stupid-cheetahs-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/vercel': patch
```

Comment:
This should be a `minor` because we are adding a new API/option

Reply:
Done.